src/Engine.sol

```diff
@@ -168,7 +180,7 @@ contract Engine is
         uint128 _accountId,
         uint256 _wordPos,
         uint256 _mask
-    ) external override {
+    ) external payable override {
```

Comment:
**Question:** I presume we have made this and other functions payable so they can be used in `MulticallablePayable` alongside a call to `fulfillOracleQuery`? The only issue I could see related to this is a future dev thinking - oh there are loads of payable functions, I can add one. But that is not a real security issue now. Can you think of any other issues that could arise from having these functions payable?

Reply:
I thought about this quite a bit and since `msg.value` is never used within any of these functions, there is no risk I am aware of by making them payable.

As a side note, I have read/heard that the `payable` func modifier might be removed in future solidity versions and every function will by default be payable. So that makes me feel better about it